frontend/sozisel-app/src/components/Files/SessionFilesParticipant/SessionFilesParticipant.tsx

```diff
@@ -0,0 +1,82 @@
+import "./SessionFilesParticipant.scss";
+
+import {
+  CircularProgress,
+  Dialog,
+  IconButton,
+  List,
+  Paper,
+  Typography,
+} from "@material-ui/core";
+
+import CloseIcon from "@material-ui/icons/Close";
+import ErrorAlert from "../../utils/Alerts/ErrorAlert";
+import FileCard from "../FileCard/FileCard";
+import { ReactElement } from "react";
+import { useParticipantFilesQuery } from "../../../graphql";
+import useSessionParticipantType from "../../../hooks/useSessionParticipantType";
+import { useTranslation } from "react-i18next";
+
+export interface SessionFilesParticipantProps {
```

Comment:
Can we change component name to `ParticipantSessionFiles`? Same goes for presenter's files. It is hard to read it the current way